gremlin-dotnet/docker-compose.yml

```
@@ -36,6 +36,12 @@ services:
       - ${HOME}/.groovy:/root/.groovy
       - ${HOME}/.m2:/root/.m2
       - ${ABS_PROJECT_HOME}/gremlin-test/target:/opt/gremlin-test
+    healthcheck:
+      test: [ "CMD-SHELL", "apk add curl && curl -f http://localhost:45940?gremlin=100-1" ]
+      interval: 30s
+      timeout: 10s
+      retries: 30
+      start_period: 1m
```

do we need 1 min wait before start pinging server container?
```suggestion
      start_period: 30s
```

---

The neo4j installation takes a bit of time, but sure